chainntnfs/txnotifier_test.go

```diff
@@ -1309,7 +1219,8 @@ func TestTxNotifierConfReorg(t *testing.T) {
 	tx1 := wire.MsgTx{Version: 1}
 	tx1.AddTxOut(&wire.TxOut{PkScript: testRawScript})
 	tx1Hash := tx1.TxHash()
-	ntfn1, err := n.RegisterConf(&tx1Hash, testRawScript, tx1NumConfs, 1)
+	ntfn1, err := n.RegisterConf(&tx1Hash, testRawScript, tx1NumConfs, 1,
```

Comment:
nit: the call parameters should be split like in the code style docs https://github.com/lightningnetwork/lnd/blob/master/docs/development_guidelines.md#wrapping-long-function-calls, later as well